xml/System/MemoryExtensions.xml

```diff
@@ -50,18 +50,9 @@
       </Parameters>
       <Docs>
         <param name="text">The target string.</param>
-        <summary>Creates a new <see cref="T:System.ReadOnlyMemory`1" /> over the portion of the target string.</summary>
-        <returns>The read-only memory representation of the string.</returns>
-        <remarks>
-          <format type="text/markdown"><![CDATA[
-
-### Remarks
-
-Returns `default` when <xref:text> is `null`.
-
-            ]]>
-          </format>
-        </remarks>
+        <summary>Creates a new <see langword="ReadOnlyMemory&lt;Char&gt;" /> over the portion of the target string.</summary>
```

Comment:
@rpetrusha I thought langword was for reserved keywords and cref was for API references. Why is langword being used here?

Reply:
The option is <cref="...ReadOnlyMemory{System.Char}"> which ends up not rendering as a link in plain text. This at least boldfaces it.

Reply:
Oh that's a neat workaround. The PR I have for System.Runtime.Intrinsics had this exact same issue and I resorted to using something like this:

> `<see cref="ReadOnlyMemory" />` for a `<see cref="System.Char" />`

 because `{System.Char}` was not working.

I'll keep your workaround in mind for the future.